src/main/groovy/com/lesfurets/jenkins/unit/declarative/ObjectUtils.groovy

```diff
@@ -3,8 +3,9 @@ package com.lesfurets.jenkins.unit.declarative
 class ObjectUtils {
 
     static String printNonNullProperties(Object obj) {
-        def props = obj.properties
+        def props = obj.properties.clone()
```

Comment:
I think we need to `.clone()` here, aren't we getting a reference in `obj.properties`?